src/git.sh

```
@@ -1,42 +1,464 @@
+  _mock_command git push origin "$version_branch"
+  
+  # Create PR - this operation is mockable
+  _mock_command gh pr create --base $merge_branch --head "$version_branch" \
+    --title "${pr_title}" --body "${pr_message}"
+    
+  # Only return the branch name
+  echo "$version_branch"
+}
+
+git_commit_version_changes() {
+  local args_json=$(parse_arguments $@")
```

Shellcheck (0.10.0)

[warning] Did you forget to close this double quoted string?

(SC1078)